### ISSUE TITLE
Error launching photo/video demo 

### ISSUE THREAD

**Issue by DG3D** [gopro/OpenGoPro#68]
**Component**
What is the bug in? 
- python demo gopro-photo and gopro-video

**Describe the bug**
Launching those demo instantly throw this error : 
``` 
% gopro-photo                            
Traceback (most recent call last):
  File "/usr/local/bin/gopro-photo", line 8, in <module>
    sys.exit(main())
TypeError: main() missing 3 required positional arguments: 'identifier', 'log_location', and 'output_location'
```

**Hardware**
- Camera: Hero9
- MacOS BigSur

**Comment by tcamise-gpsw** [comment]
Sorry about that. I've been testing by calling main directly so I missed that. I'll push out an updated open-gopro package tomorrow.

**Comment by tcamise-gpsw** [comment]
Ok I've published version 0.6.2 which should have this fixed now
